captum/optim/models/_image/inception_v1.py

```diff
@@ -225,7 +232,7 @@ def __init__(
         c5x5reduce: int,
         c5x5: int,
         pool_proj: int,
-        activ=ReluLayer,
+        activ=nn.ReLU,
         p_layer=nn.MaxPool2d,
```

Comment:
Suggestion: These should probably be type hinted as `Type[nn.Module]`

Reply:
Done!